.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5503
+}
```

Comment:
Can we remove the `.vscode` directory from the repository? Maybe add it to your `.gitignore`